c_models/makefiles/my_if_curr_exp_sEMD_neuron/Makefile

```diff
@@ -0,0 +1,6 @@
+APP = $(notdir $(CURDIR))
+
+OTHER_SOURCES = $(NEURON_DIR)/neuron/models/neuron_model_lif_impl.c
+NEURON_IMPL_H = $(EXTRA_SRC_DIR)/my_models/implementations/my_neuron_impl_semd.h
```

Comment:
I wasn't aware it was possible to do a makefile this way, but if it works, happy to keep it as an example...

Edit: Ah, I see how it works now, the NEURON_IMPL_H file includes the actual threshold type etc. that it uses directly.  It was confusing me for a minute, so I'll leave this comment here just in case it confuses anyone else in the future.

Reply:
To be clear, this is the same Makefile as is used for sEMD with the combined synapse-neuron model, except the synapse bit has been removed, and the include is neuron_build.mk instead of neural_build.mk.  Again for reference, this is an easy way to extract the neuron bit from a neural build.